src/scripts/libs/snippets.js

```diff
@@ -0,0 +1,3 @@
+// from Material Design Lite website
```

Comment:
@gauntface 

Just to confirm

  1. If it is only to allow eslinting, I can rewrite the material design code, what do you say?
  2. if you want to use your code―I was actually using this very code from web fundamentals―do you want me to call it remotely (otherwise I would not understand why you would talk about the module thing as I could just copy the code inside it). Although I think calling it remotely would create a hard to maintain dependency.

Anyway, let's do it your way, yes! Just answer the above and I will make my move.